autogen

```diff
@@ -27,6 +28,67 @@ autoconf --force --warnings=all,error
 sed -e '/^runstatedir/d' \
     -e '/-runstatedir /{N;N;N;N;N;N;N;N;d;}' \
     -e '/--runstatedir=DIR/d' \
-    -e 's/ runstatedir//' configure > configure.tmp
-mv -f configure.tmp configure
+    -e 's/ runstatedir//' \
+    -e '1d' configure > configure.tmp
```

Comment:
nitpick: if `> configure.tmp` was on its own line, it wouldn't be part of diffs on changes such as this one.

Reply:
Hah - I use `patdiff` which already doesn't include that in the `diff`, but the "code" is better with the extra line break, I agree.